oletools/msodde.py

```diff
@@ -67,10 +67,14 @@
 
 # little hack to allow absolute imports even if oletools is not installed
 # Copied from olevba.py
-PARENT_DIR = dirname(dirname(abspath(__file__)))
-if PARENT_DIR not in sys.path:
-    sys.path.insert(0, PARENT_DIR)
-del PARENT_DIR
+try:
+    from oletools.thirdparty import olefile
+except ImportError:
+    PARENT_DIR = dirname(dirname(abspath(__file__)))
+    if PARENT_DIR not in sys.path:
+        sys.path.insert(0, PARENT_DIR)
+    del PARENT_DIR
+    from oletools.thirdparty import olefile
 
 from oletools.thirdparty import olefile
```

Comment:
this one is redundant, can be removed.

Reply:
I guess you mean the one on line 75/79? I'll send you another pull request for this